Packs/CommonScripts/Scripts/GetDomainCertificate/README.md

```diff
@@ -0,0 +1,48 @@
+# GetDomainCertificate
+
+## Script Data
+---
+
+| **Name** | **Description** |
+| --- | --- |
+| Script Type | python3 |
+| Tags | certificate, domain |
+
+## Inputs
+---
+
+| **Argument Name** | **Description** |
+| --- | --- |
+| domain | The domain to retrieve the certificate for |
```

Comment:
```suggestion
| domain | The domain to retrieve the certificate for. |
```
This does not match what is in the yml file.  The argument should be "domains" and the description should be the same as the yml file.